packages/fxa-auth-server/test/local/routes/subscriptions/apple.js

```diff
@@ -0,0 +1,160 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const sinon = require('sinon');
+const { default: Container } = require('typedi');
+const assert = { ...sinon.assert, ...require('chai').assert };
+const uuid = require('uuid');
+
+const mocks = require('../../../mocks');
+const {
+  AppleIapHandler,
+} = require('../../../../lib/routes/subscriptions/apple');
+const {
+  PurchaseUpdateError,
+} = require('../../../../lib/payments/iap/apple-app-store/types/errors');
+const error = require('../../../../lib/error');
+const { AuthLogger } = require('../../../../lib/types');
+const {
+  AppleIAP,
+} = require('../../../../lib/payments/iap/apple-app-store/apple-iap');
+const { IAPConfig } = require('../../../../lib/payments/iap/iap-config');
+const { OAUTH_SCOPE_SUBSCRIPTIONS_IAP } = require('fxa-shared/oauth/constants');
+const { CapabilityService } = require('../../../../lib/payments/capability');
+
+const MOCK_SCOPES = ['profile:email', OAUTH_SCOPE_SUBSCRIPTIONS_IAP];
```

Comment:
Do we need the `'profile:email'` scope? It's not used in the handler AFAICT.

Reply:
Nope.